perceval/converters/abstract_converter.py

```diff
@@ -77,6 +77,18 @@ def _configure_processor(self, gate_circuit, **kwargs):
             self._converted_processor.add_port(i * 2, Port(Encoding.DUAL_RAIL, f'{qname}{i}'))
             self._input_list[i * 2] = 1
 
+    def _configure_processor_from_nqubits(self, n_qbits, **kwargs):
+        """
```

Comment:
`_configure_processor_from_nqubits` method is identical in action to `_configure_processor` except the required input parameter `n_qubits`. The existing method calls `count_qubits` to get the value. I would suggest using it for the new QASMConverter class as well.